paddle/capi/examples/model_inference/dense/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ project(dense)
 cmake_minimum_required(VERSION 2.8)
 aux_source_directory(. SRC_LIST)
 add_executable(${PROJECT_NAME} ${SRC_LIST})
-set_property(TARGET ${PROJECT_NAME} PROPERTY C_STANDARD 99)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
All the examples are written in C style. Maybe we'd better not change this but remove the C++ related codes.

Reply:
Done